src/gsys_pipeline.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+from log_parser import EventRule
+from event_parser import EventGroupRule
+from signals import *
```

Comment:
In general "import *" imports are frowned upon. They tend to produce a lot of namespace pollution, and they can make things more brittle where introducing new symbols into the imported-from module can accidentally break modules pulling them in with * imports.

In fact, using from-style imports in general is against the Google style; but the * import was much more egregious.

Reply:
Would it be best to change them all to import <module> and <module>.<thing> usage?

Reply:
Wow the formatting butchered that comment...
"import module"
and 
"module.thing"

Reply:
Ideally converting everything to drop the "from ... import" style would be good. But if you want to do that it might be better to just leave it for latter on in a future commit. For this I'd just avoid using * imports.

Reply:
ok, ill fix it to be not the * for this one and then spin a PR after this one to fix all the imports.